pkg/controller/ingress-shim/controller.go

```diff
@@ -96,7 +84,6 @@ func (c *controller) Register(ctx *controllerpkg.Context) (workqueue.RateLimitin
 		WorkFunc: certificateDeleted(queue),
 	})
 
-	c.helper = issuer.NewHelper(c.issuerLister, c.clusterIssuerLister)
```

Comment:
Question: was this helper unused?

Reply:
Yes, I think it was a leftover from a copy-paste of the Issuer controller